NuGetReleaseTool/NuGetReleaseTool/NuGetReleaseTool.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net7.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="CommandLineParser" Version="2.7.82" />
```

Comment:
Why not System.CommandLine?

Reply:
The previous projects were using CommandLineParser so I just kept it. 
It was just easier cause the work was already done.

Reply:
I have no problem with it using System.Commandline eventually. 
As someone who has experience with System.Commandline, feel free to make a change later on. 

I don't feel too strongly either way.